### ISSUE TITLE
Infinite scroll doesn't apply to custom row items

### ISSUE THREAD

**Issue by taxigy** [GriddleGriddle/Griddle#257]
I have a custom row item that looks pretty similar to [one in docs](http://griddlegriddle.github.io/Griddle/customization.html#custom-row-format), and I want to apply infinite scroll to the grid. It doesn't work. As I scroll down to the last line of currently shown items, nothing happens.

I put the grid this way:

```
<Griddle
  results={debugMessages}
  columns={['title', 'date', 'type']}
  enableInfiniteScroll={true}
  bodyHeight={400}
  resultsPerPage={debugMessages.length / 2}
  useGriddleStyles={false}
  useCustomRowComponent={true}
  customRowComponent={debugItem} />
```

and the item element looks like this:

```
class debugItem extends Component {
  render() {
    var containerStyle = {
      'float': 'left',
      'width': '33%',
      'margin-bottom': '25px'
    };

    var imageStyle = {
      'float': 'left',
      'height': '80px',
      'width': '80px',
      'margin-right': '25px'
    };

    var headerStyle = {
      'font-size': '1em',
      'color': 'rgb(30, 30, 30)',
      'margin-bottom': '0.25em'
    };

    var paragraphStyle = {
      'font-size': '0.8em',
      'color': 'rgb(60, 60, 60)',
      'margin-bottom': '0.15em'
    };

    return (
      <div style={containerStyle}>
        <img src={this.props.data.imageUrl} style={imageStyle} />
        <h3 style={headerStyle}>{this.props.data.title}</h3>
        <p style={paragraphStyle}>{this.props.data.type} {this.props.data.size}</p>
        <p style={paragraphStyle}>As of {this.props.data.date}</p>
      </div>
    );
  }
}
```

What do I do wrong? Does Griddle even allow custom row components scrolled infinitely?

**Comment by jgwconsulting** [comment]
Yes, I couldn't get it to work either. I guess it's not supported currently. 

However, there are a couple of other React (non-grid based) infinite scroll components. This one looks good, although I haven't tried it: https://github.com/orgsync/react-list. Slightly odd how renderItem is specified rather than using a component but I guess it works fine.

**Comment by CrossfireCurt** [comment]
+1 very interested in the answer to this too.

**Comment by legomind** [comment]
@taxigy Try setting an absolute height to your row style and setting the `rowHeight` prop on your griddle instance.

**Comment by michaelhartmayer** [comment]
Has there been any new development on this?

**Comment by barnaby** [comment]
+1

**Comment by abologna-r7** [comment]
still nothing on this?

**Comment by ryanlanciaux** [comment]
For now, this is by design. The infinite scrolling is designed to work for `<tr>` tags. We have recently released 1.0 which has a plugin model. We'd love if someone wanted to try to make a plugin for this sort of interaction. Please [see the plugin documentation for information on creating plugins](http://griddlegriddle.github.io/Griddle/docs/plugins/) if there is any interest in this. We have a [virtual scrolling plugin in Griddle 1.0](https://github.com/GriddleGriddle/Griddle/tree/master/src/plugins/position) that could be extended or used as a reference. 

I'm closing this one for now but please feel free to open another issue if any problems are encountered. 👍
